### PR TITLE
adding various responses

### PR DESCRIPTION
**Pull request by vekst**
hello

### CODE REVIEW

File: cogs/Fun/8ball/8ball.json [hunk 1]
```diff
@@ -20,6 +20,16 @@
     "My sources say no.",
     "Outlook not so good.",
     "Very doubtful.",
-    "lolno."
+    "lolno.",
+    "Who do you think I am?",
+    "The answer is obvious, of course.",
+    "Not in a million years.",
+    "If you try hard enough, perhaps."
```
Comment (Snaptraks) [inline cogs/Fun/8ball/8ball.json]: needs a `,` at the end of this line

File: cogs/Responses/mentions.json [hunk 2]
```diff
@@ -14,6 +14,11 @@
     "I'm not bad I'm just coded that way.",
     "Code me like one of your optimization algorithms.",
     "Ask <@337266376941240320> instead.",
-    "They say all the best things in life are free, so give all your <:feesh:427018890137174016> and your code to me!"
+    "They say all the best things in life are free, so give all your <:feesh:427018890137174016> and your code to me!",
+    "Who are you and what do you want?!",
+    "Psssst, wanna know a secret? Xplio smells!",
```
Comment (Snaptraks) [inline cogs/Responses/mentions.json]: maybe use lower case xplio? he never uses Xplio

Comment (vekst) [reply cogs/Responses/mentions.json]: Xplio just made sense since it's the start of a sentence, I thought about just xploio a lot

**Comment by Snaptraks** [comment]
Everything is working, I will merge!

**Pull request merged**